package.json

```diff
@@ -86,10 +87,11 @@
   },
   "scripts": {
     "pretest": "npm run build",
-    "test": "npm run check:node && npm run check:web",
+    "test": "npm run check:node && npm run check:web && npm run check:web-no-bigint",
```

Comment:
Only testing for web without bigint here as we expect that Node will always support BigInt. Open to changing this if anyone has other thoughts. (@nbbeeken, @addaleax)

Reply:
The codepaths are largely the same (we mostly take a different path for `Buffer` related things).  I think this is sufficient.  I'm mainly concerned with the core serialization / deserialization / EJSON.stringification / EJSON.parsing, which is mostly the same across Node and web.